qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -211,12 +213,19 @@ def calculate_1q_epg(
                     found_gates.append(gate)
                     error_sum += gates_per_clifford[key] * value
             for gate in found_gates:
-                epg[(qubit,)][gate] = (gate_error_ratio[((qubit,), gate)] * epc_1_qubit) / error_sum
+                epg_value = (gate_error_ratio[((qubit,), gate)] * epc_1_qubit) / error_sum
```

Comment:
This `error_sum` is error-free value right?

Reply:
Yes, the values we get from the backends are (currently?) without stderr.